keyboards/doio/kb12/kb12.c

```diff
@@ -0,0 +1,47 @@
+/* Copyright 2022 DOIO
+ * Copyright 2022 DOIO2022 <https://github.com/DOIO2022>
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "quantum.h"
+
+
+#ifdef RGB_MATRIX_ENABLE
+
+ led_config_t g_led_config = { {
```

Comment:
This can be expressed in info.json, in which the file can potentially be removed.

Reply:
@zvecr Hi, thank you for your changes ,  but  I don't understand about this change. Could you  give me an example? Thank you so much!

Reply:
`qmk migrate` or `qmk info -kb (name) -f json` should help with that.